examples/calendar/backend/services/appointments/controller.ts

```diff
@@ -0,0 +1,36 @@
+import { NextFunction, Request, Response } from "express";
+import { Appointment } from "../../models";
+import { ethers, BigNumber } from "ethers";
+
+export const create = async (req: Request, res: Response, next: NextFunction) => {
+	try {
+		const secret = BigNumber.from(ethers.utils.randomBytes(Math.floor(Math.random() * 100)));
+		const userId = (req as any).user.payload.id;
+		const appointmentObject = {
+			fromUser: userId,
+			status: "created",
+			secret: secret.toString()
+		};
+		const appointment = await Appointment.create(appointmentObject);
+		const result = {appointment: appointment};
```

Comment:
const result = {appointment}

Reply:
I just wrote the frontend like this, I know it is minor issue but I hope we can let it pass?